shared/ui/Stream/PullRequests/Bitbucket/PullRequestConversationTab.tsx

```diff
@@ -0,0 +1,303 @@
+import { BitbucketParticipantEditScreen } from "../Bitbucket/BitbucketParticipantEditScreen";
+
+// const emojiMap: { [key: string]: string } = require("../../agent/emoji/emojis.json");
+// const emojiRegex = /:([-+_a-z0-9]+):/g;
+
+export const Circle = styled.div`
+	width: 12px;
+	height: 12px;
+	border-radius: 6px;
+	display: inline-block;
+	margin-right: 5px;
+	vertical-align: -1px;
+`;
+
+const UL = styled.ul`
+	padding-left: 20px;
+	li {
+		margin: 5px 0;
+	}
+	margin: 20px 0;
+`;
+
+const StatusMetaRow = styled.div`
+	flex: 1;
+	.titleRow {
+		display: flex;
+	}
+	.middle {
+		flex: 1;
+	}
+	.checkRuns {
+		margin: 10px -15px 0 -15px;
+		max-height: 0;
+		overflow-y: auto;
+		transition: max-height 0.2s;
+		&.expanded {
+			max-height: 231px;
+		}
+		.checkRun {
+			display: flex;
+			align-items: center;
+			border-top: 1px solid;
+			border-color: var(--base-border-color);
+			padding: 3px 15px;
+			${PRIconButton} {
+				margin-right: 10px;
+			}
+			.details {
+				margin-left: 10px;
+			}
+			.description {
+				flex: 1;
+				display: block;
+			}
+			.appIcon {
+				display: block;
+				margin-right: 8px;
+				width: 20px;
+				height: 20px;
+				border-radius: 6px;
+				overflow: hidden;
+				background-color: #fff;
+				&.appIconFallback {
+					color: #678;
+					display: flex;
+					align-items: center;
+					justify-content: center;
+				}
+			}
+		}
+	}
+`;
+
+const EMPTY_HASH = {};
+const EMPTY_ARRAY = [];
+let insertText;
+let insertNewline;
+let focusOnMessageInput;
+
+export const PullRequestConversationTab = (props: {
+	setIsLoadingMessage: Function;
+	bbRepo: any;
+	checkMergeabilityStatus: Function;
+	autoCheckedMergeability: autoCheckedMergeabilityStatus;
+	initialScrollPosition: number;
+}) => {
+	const { bbRepo, setIsLoadingMessage } = props;
+	const dispatch = useAppDispatch();
+	const derivedState = useAppSelector((state: CodeStreamState) => {
+		const currentUser = state.users[state.session.userId!] as CSMe;
+		const team = state.teams[state.context.currentTeamId];
+		const blameMap = team.settings ? team.settings.blameMap : EMPTY_HASH;
+		const skipGitEmailCheck = state.preferences.skipGitEmailCheck;
+		const addBlameMapEnabled = isFeatureEnabled(state, "addBlameMap");
+		const currentPullRequest = getCurrentProviderPullRequest(state);
+		const { preferences, ide } = state;
+
+		const prConnectedProviders = providerSelectors.getConnectedSupportedPullRequestHosts(state);
+
+		return {
+			defaultMergeMethod: preferences.lastPRMergeMethod || "SQUASH",
+			currentUser,
+			currentPullRequestId: state.context.currentPullRequest
+				? state.context.currentPullRequest.id
+				: undefined,
+			blameMap,
+			currentPullRequest: currentPullRequest,
+			currentPullRequestProviderId: state.context.currentPullRequest
+				? state.context.currentPullRequest.providerId
+				: undefined,
+			pr: currentPullRequest?.conversations?.repository?.pullRequest! as any, // TODO handle undefined / null
```

Comment:
Better temporary fix - keep the type

```suggestion
    pr: currentPullRequest?.conversations?.repository?.pullRequest!,
```